packages/web-components/src/components/footer/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,134 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import corsproxy from '../../../../../../docs/cors-proxy.md';
+
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+<Preview withToolbar>
+  <Story id="components-footer--default" height="720px" />
+</Preview>
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/footer)
+> example implementation.
+
+## Getting started
+
+##### JS
+
+```javascript
+import ConnectMixin from '@carbon/ibmdotcom-web-components/es/globals/mixins/connect';
+import DDSFooterComposite from '@carbon/ibmdotcom-web-components/es/components/footer/footer-composite';
+import { store, mapStateToProps, mapDispatchToProps } from '@carbon/ibmdotcom-web-components/es/components/footer/footer-container';
+
+// Wires `<dds-footer-composite>` to `@carbon/ibmdotcom-servies` and creates `<app-footer-container>`
+customElements.define('app-footer-container', class extends ConnectMixin(store, mapStateToProps, mapDispatchToProps)(DDSFooterComposite) {})
+```
+
+##### HTML
+
+```html
+<app-footer-container></app-footer-container>
+```
+
+## `<dds-footer-composite>`
+
+`<dds-footer-composite>` automatically renders contents of footer from the given data.
+You can populate the data automatically from `@carbon/ibmdotcom-servies` by mixing a Redux store into this custom element class, shown in the example in Getting started section above.
```

Comment:
We probably should simplify this (and in general all of the components we are building out), and maybe have two different versions of what most adopters will want to do (ELI5, I just need one import and drop a tag on the page and call it a day), or the adopters that do care more about what's happening under the hood.

So one version I imagine would be what most adopters would be, and just want something like:

```
## Getting Started

### JS

import '@carbon/ibmdotcom-web-components/es/components/footer/footer'; (or whatever the import is)

### HTML

<dds-footer></dds-footer>
```

Then there would be the one you might have above if they want to get a little fancier with connecting data, etc:

```
### Under the hood

... explain what imports there are and how everything is getting connected ...
```

We're going to encounter that most adopters barely can pull together an HTML page, let alone understanding Typescript, etc and trying to connect one of the primary components for IBM.com on the page like this would be a huge challenge for them.

Reply:
Thank you for your feedback @jeffchew - Updated as discussed offline.